internal/pkg/consensus/processor.go

```diff
@@ -173,6 +116,10 @@ func (p *DefaultProcessor) ProcessTipSet(ctx context.Context, st state.Tree, vms
 	}
 	bh := types.NewBlockHeight(h)
 
+	var res ProcessTipSetResponse
+	res.Failures = make(map[cid.Cid]struct{})
+	res.Successes = make(map[cid.Cid]struct{})
```

Comment:
This method has a named return value `tsResult` - use that one.

Reply:
good catch.